app/controller/comments-controller.js

```diff
@@ -0,0 +1,15 @@
+const { checkArticleExists } = require("../model/check-articles");
+const { selectComments } = require("../model/select-comments");
+
+exports.getComments = (req, res, next) => {
+  const { article_id } = req.params;
+  Promise.all([selectComments(article_id), checkArticleExists(article_id)])
```

Comment:
Great that you've implemented this to check the article exists, just need a test to check this functionality! 🧪